google-cloud-firestore/src/main/java/com/google/cloud/firestore/Transaction.java

```diff
@@ -38,6 +38,10 @@
  */
 public final class Transaction extends UpdateBuilder<Transaction> {
 
+  Transaction wrapResult(ApiFuture<WriteResult> result) {
+    return this;
+  }
+
```

Comment:
nit: package-public method are generally found after public method in a class, not before the static fields.

Reply:
Moved further down.